src/planning.py

```diff
@@ -423,6 +424,52 @@ def _run_low_level_search(task: Task, option_model: _OptionModelBase,
     return [], True
 
 
+def _run_plan_with_option_model(
+        task: Task, task_idx: int, option_model: _OptionModelBase,
+        plan: List[_Option]) -> Tuple[LowLevelTrajectory, bool]:
+    """Runs a plan on an option model to generate a low level trajectory.
+
+    Returns a LowLevelTrajectory and a boolean. If the boolean is True,
+    the option sequence successfully executed to achieve the goal and
+    generated a LowLevelTrajectory. Otherwise, it returns an empty list
+    and False.
+    """
+    traj: List[State] = [task.init] + [DefaultState for _ in plan]
```

Comment:
it's fine to leave these type annotations, but I would be surprised if they're needed, since mypy should be able to figure out the types from the initialization